integrators/ordinary_differential_equations_body.hpp

```diff
@@ -1,16 +1,33 @@
 ﻿
 #pragma once
 
+#include "integrators/ordinary_differential_equations.hpp"
+
+#include <vector>
+
 #include "base/macros.hpp"
 #include "integrators/embedded_explicit_runge_kutta_nyström_integrator.hpp"
-#include "integrators/ordinary_differential_equations.hpp"
 #include "integrators/symmetric_linear_multistep_integrator.hpp"
 #include "integrators/symplectic_runge_kutta_nyström_integrator.hpp"
 
 namespace principia {
 namespace integrators {
 namespace internal_ordinary_differential_equations {
 
+template<typename Position_>
+SpecialSecondOrderDifferentialEquation<Position_>::SystemState::SystemState(
+    std::vector<typename SpecialSecondOrderDifferentialEquation<
+        Position_>::Position> const& q,
```

Comment:
Can't we just write `std::vector<Position>` here?  I would expect that name to be visible.